src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -8,40 +9,273 @@
+                                                                     "checked=\"checked\"" :
+                                                                      "";
+        
+        newForm.responseVisibleNeverCheckedAttribute = (newFeedbackSession != null 
+                                                        && Const.TIME_REPRESENTS_NEVER
+                                                           .equals(newFeedbackSession.resultsVisibleFromTime)) ?
+                                                                        "checked=\"checked\"" : "";
+                                
+        newForm.submitButtonDisabledAttribute = courses.isEmpty() ? " disabled=\"disabled\"" : "";
+   
+    }
+    
+    
+    List<FeedbackSessionRow> convertFeedbackSessionAttributesToSessionRows(List<FeedbackSessionAttributes> sessions, 
+                                    HashMap<String, InstructorAttributes> instructors, String feedbackSessionNameForSessionList, String courseIdForNewSession) {
+
+        
+        List<FeedbackSessionRow> rows = new ArrayList<FeedbackSessionRow>();
+        int displayedStatsCount = 0;
+        
+        for (FeedbackSessionAttributes session : sessions) {
+            String courseId = session.courseId;
+            String name = PageData.sanitizeForHtml(session.feedbackSessionName);
+            String tooltip = PageData.getInstructorHoverMessageForFeedbackSession(session);
+            String status = PageData.getInstructorStatusForFeedbackSession(session);
+            String href = getFeedbackSessionStatsLink(session.courseId, session.feedbackSessionName);
+            
+            String recent = "";
+            if (session.isOpened() || session.isWaitingToOpen()) {
+                recent = " recent";
+            } else if (displayedStatsCount < InstructorFeedbacksPageData.MAX_CLOSED_SESSION_STATS
+                       && !TimeHelper.isOlderThanAYear(session.createdTime)) {
+                recent = " recent";
+                ++displayedStatsCount;
+            }
+            
+            String actions = "";
+            try {
+                actions = getInstructorFeedbackSessionActions(session, false, instructors.get(courseId),
+                                                getCourseIdSectionNamesMap(sessions).get(courseId));
+            } catch (EntityDoesNotExistException e) {
+                // nothing
+            }
+            
+            ElementTag elementAttributes ;
+            if (session.courseId.equals(courseIdForNewSession) && session.feedbackSessionName.equals(feedbackSessionNameForSessionList)) {
+                elementAttributes = new ElementTag("class", "sessionsRow warning");
+            } else {
+                elementAttributes = new ElementTag("class", "sessionsRow");
+            }
+            
+            rows.add(new FeedbackSessionRow(courseId, name, tooltip, status, href, recent, actions, elementAttributes));
```

Comment:
If there is only one attribute in elementAttributes, why not use a single String?